io_scene_nif/modules/animation/animation_import.py

```diff
@@ -236,7 +235,7 @@ def set_frames_per_second(self, roots):
 class ObjectAnimation:
 
     def __init__(self, parent):
-        self.nif_import = parent
+        self.animationhelper = parent
```

Comment:
There is no parent reference being passed in, is this needed?

Reply:
We don't need nif_import but need to access functions on the Animation class. Not all functions of Animation are staticmethod and it's in the same file, so no import. I think Animation is only used to store FPS so we can probably clean it up further

Reply:
I wonder if we should have Animation as an ABC then.
Yeah, would be good to have all of these constants using or exposed as scene properties.
For now, localising to the module they are actually required for/should be owned by should be enough

Reply:
ABC?

Right, FPS should be a scene thing. It is in blender, anyway. We ended up storing it in a class because setting / getting it from blender caused trouble. Was probably missing an update() on the blender scene back then.

Reply:
An abstract base case class, basically inheritance.